__tests__/app.test.js

```diff
@@ -93,9 +94,27 @@ describe("/api/articles/:article_id", () => {
   });
 });
 
-  
-  
+describe('/api/articles', () => {
+  test('GET:200 - responds with all the articles', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toHaveLength(13);
 
-  
+        body.articles.forEach(article => {
+          expect(Object.keys(article).length).toBeGreaterThan(0); 
+          expect(typeof article.author).toBe('string');
+          expect(typeof article.title).toBe('string');
+          expect(typeof article.article_id).toBe('number');
+          expect(typeof article.topic).toBe('string');
+          expect(typeof article.created_at).toBe('string');
+          expect(typeof article.votes).toBe('number');
+          expect(typeof article.article_img_url).toBe('string');
+          expect(typeof article.comment_count).toBe('string');
```

Comment:
It would be good if this was a number, you can either `CAST` the comment count in sql or parse it in javascript either way would work

Reply:
so should i do in my db query something like COUNT(comments.article_id) AS CAST(comment_count AS int) ?